pkg/distributor/otlp/timeseries.go

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+//go:generate ./generate.sh
+
+package otlp
+
+import (
+	"github.com/grafana/mimir/pkg/mimirpb"
```

Comment:
Shall we add

```
	// Dummy dependency to ensure these files get vendored and we can use them as transformation sources.
 	_ "github.com/prometheus/prometheus/storage/remote/otlptranslator/prometheusremotewrite"
```

here?

Reply:
Yep, I just realized I forgot to do this.

Reply:
Done.